src/components/ClickWrapper/index.jsx

```diff
@@ -143,8 +150,12 @@ ClickWrapper.defaultProps = {
 
 
 CallbackWrapper.propTypes = {
```

Comment:
I think we only need to add descriptions on the root components inside `index.jsx` and `Markup.jsx` files.

Reply:
Makes sense, updated as requested